actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -95,7 +95,11 @@ def tag_option(key, value, escape)
 
         private
           def prefix_tag_option(prefix, key, value, escape)
-            key = "#{prefix}-#{key.to_s.dasherize}"
+            if prefix == 'data' and key == :disable_with
```

Comment:
The page I've been profiling on only has this option. @rafaelfranca do you know how to find other common options to add to this?